_includes/policy_nav.html

```diff
@@ -0,0 +1,24 @@
+<nav id="pb-nav--side-cntnr" class="sm-col-right sm-col-4 sm-show policy">
+  <ul id="pb-nav--side" class="list-reset nav">
+    {% assign section_index = 0 %}
+    {% for section in site.translations[site.lang]["policies"]["sections"] %}
+      <li class="border-bottom nav-sidenav-item">
+        <a href="{{ section.link.url | prepend: site.baseurl }}" class="p2 block h6">
```

Comment:
@slj could this still be generating a URL with a domain in it?

Reply:
Based on the Fed docs, I say no.

Reply:
With the code like this, the Fed build paths are working.  The only problem now is the external link test script failures.  The problem seems to be due to the script trying to test files that are not live (ie merged to master).  Updating the script with some of the code from https://github.com/gjtorikian/html-proofer#ignoring-new-files, the local tests passed except on `how-does-it-work`?  No idea right now why...

Reply:
hmmmmmm....this is probably because the `how-does-it-work` link was established in a previous commit, so the code snippet doesn't think it's new...maybe if I rebase it so the diff is one level deep it might work...fun lol!

Reply:
Local test on squashed commit still failed on that same directory `how-does-it-work`.  SMH...I may need to abort this PR...

Reply:
> Ok so I just ran this locally, the URLs its picking up are from the `<link rel="canonical" href="...." />` tags, I'm going to see if there's tooling we can add to ignore just `<link>` tags

Thanks for looking into this w/me.  What clued you into the URLs coming from there?